### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by NicoLeOca** [Kane610/aiounifi#235] [General]
Hi,

I noticed your commit in the new release of HA and I just pass by to thank you for improving Unifi integration in HA.
I am just a commited user, sad to see too little development around this integration.

Cheers

NicoLeOca

**Comment by Kane610** [comment]
Thank you! Positive feedback are rare to receive!

**Comment by NicoLeOca** [comment]
Again, I see your name in the HA changelog! 👍 
Thank you very much for impleting the POE power usage into HA. I spent hours yesterday to check how to do it.
your work is much appreciated
